crates/hc_bundle/src/cli.rs

```diff
@@ -211,15 +229,27 @@ impl HcDnaBundle {
             Self::Unpack {
                 path,
                 output,
+                raw,
                 force,
             } => {
-                let dir_path = crate::packing::unpack::<ValidatedDnaManifest>(
-                    DNA_BUNDLE_EXT,
-                    &path,
-                    output,
-                    force,
-                )
-                .await?;
+                let dir_path = if raw {
+                    crate::packing::unpack_raw(
+                        DNA_BUNDLE_EXT,
+                        &path,
+                        output,
+                        ValidatedDnaManifest::path().as_ref(),
+                        force,
+                    )
+                    .await?
+                } else {
+                    crate::packing::unpack::<ValidatedDnaManifest>(
+                        DNA_BUNDLE_EXT,
+                        &path,
+                        output,
+                        force,
+                    )
+                    .await?
+                };
                 println!("Unpacked to directory {}", dir_path.to_string_lossy());
```

Comment:
In the case of `dna unpack`, the message could say "Manifest extracted..", so that the user doesn't expect other files like wasms.

Reply:
The wasms weren't unpacked in my test, hence the suggestion.